lua/nvim-treesitter/textobjects/shared.lua

```diff
@@ -86,6 +89,20 @@ function M.textobject_at_point(query_string, pos, bufnr, opts)
           lookahead_earliest_start = start_byte
         end
       end
+    elseif opts.lookbehind then
```

Comment:
it appear that throught the `else` lookahead and lookbehind are exclusive. I don't think they need to be. E.g. when nothing is found lookahead, it could look behind. If they are intended to be exclusive it should also be only one config key to configure them, but I'd perfer to allow to have them both.

Reply:
It's not intended. I changed it.